ansible/templates/omero-ms-zarr-entrypoint.sh.j2

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -eu
+
+grep omerozarr /etc/group || addgroup -g {{ idrnfs_groupid | default(1000) }} omerozarr
```

Comment:
This is needed due to how NFS is setup

Reply:
Might be worth capturing this and other tidbits in the PR itself.